src/components/MapPage/index.js

```diff
@@ -6,7 +6,18 @@ const MapPage = () => {
   const [canEnterAddress, setCanEnterAddress] = useState(false);
   const [userAddress, setUserAddress] = useState('');
   const [userLocation, setUserLocation] = useState([]);
-  const [locations, setLocations] = useState([]);
+  const [closestLocation, setClosestLocation] = useState({
+      "id": 1,
+      "name": "Denver Walk-In Crisis Services",
+      "phone": 3035041299,
+      "address": "4353 E. Colfax Avenue",
+      "city": "Denver", 
+      "state": "CO", 
+      "zip": 80220,
+      "hours": "24 hours a day, 7 days a week",
+      "lat": 39.7403,
+      "lng": -104.9363
+    });
 
```

Comment:
I'm digging the hooks.  I need to practice writing them myself!  I'm assuming this placeholder will get replaced with the result of the fetch for closest location.